package.json

```diff
@@ -47,8 +47,10 @@
     "angular-resource": "1.6.9",
     "angular-route": "1.6.9",
     "angular-vs-repeat": "1.1.7",
+    "body-parser": "1.18.3",
```

Comment:
I forgot to mention this in previous comments. Can you remove 'body-parser' and 'cors' from package.json? They aren't really used anywhere but in a test server which is itself is only for reference, so you can comment the entire file out if typescript is complaining about these dependencies.

Reply:
okay I'll just put the example in javascript instead as it is not using any typescript feature